step2/chat/ChatController.js

```diff
@@ -0,0 +1,12 @@
+const storage = new(require('../dataStores/Storage'));
```

Comment:
zastanawiam się czy podobne do tego konstrukcje to nie jest jakiś antypattern - znaczy jak się zapomni o tym zewnętrznym nawiasie to przestanie działać. Może lepiej to rozbić na dwie linie?